src/driver/kvm.rs

```diff
@@ -87,12 +121,38 @@ impl<T: KVMIntrospectable> Introspectable for Kvm<T> {
             rip: regs.rip,
             rflags: regs.rflags,
             cr0: sregs.cr0,
+            cr2: sregs.cr2,
             cr3: sregs.cr3,
             cr4: sregs.cr4,
-            fs_base: sregs.fs.base,
+            sysenter_cs: _msrs.entries[0].data,
+            sysenter_esp: _msrs.entries[1].data,
+            sysenter_eip: _msrs.entries[2].data,
+            msr_efer: _msrs.entries[3].data,
+            msr_star: _msrs.entries[4].data,
+            msr_lstar: _msrs.entries[5].data,
+            efer: sregs.efer,
+            apic_base: sregs.apic_base,
+            cs: SegmentReg::from(sregs.cs),
```

Comment:
That looks so much nicer already :+1: But if you implement `From` you get `Into` for free, so that means you can just write:
```suggestion
            cs: sregs.cs.into(),
```